Makefile

```diff
@@ -27,6 +27,7 @@ override IMAGE_VERSION := e2e-$(GIT_COMMIT)
 endif
 endif
 IMAGE_TAG ?= $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_VERSION)
+RELEASE_TAG ?= $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_VERSION)
```

Comment:
why create a new tag var here? I think we need to fix this PR: https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/585, it has already set `os.version`, while there is sth. wrong

Reply:
also should have an test to prevent such regression in the future.

Reply:
got it, that's for latstet tag